src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/EditAndContinueTests.cs

```diff
@@ -2079,7 +2079,7 @@ interface J { }
 
             CheckEncLog(reader2,
                 Row(3, TableIndex.AssemblyRef, EditAndContinueOperation.Default),
-                Row(8, TableIndex.TypeRef, EditAndContinueOperation.Default),
+                Row(10, TableIndex.TypeRef, EditAndContinueOperation.Default),
```

Comment:
@tmat wanted to make sure you saw this change. This change just moved from the netcoreapp3.1 to net5.0 reference assemblies. Unsure why this would cause the diff to change here. This is the only one that changed though.

Reply:
The TypeRef is likely System.Action. Are we emitting now 2 more TypeRefs into each assembly?

---
In reply to: [562016108](https://github.com/dotnet/roslyn/pull/50486#discussion_r562016108) [](ancestors = 562016108)